tests/publiccloud/ipa.pm

```diff
@@ -29,6 +29,14 @@ sub run {
         results_dir => 'ipa_results'
     );
 
+    if (get_var('PUBLIC_CLOUD_CHECK_BOOT_TIME')) {
+        my $kernel_max_boot_time = 60;
+        my $system_max_boot_time = 120;
+        my $out                  = script_output('grep "^Startup finished in" ' . $ipa->{logfile});
+        die 'Fail to find boot time in log' unless $out =~ /Startup finished in (\d{1,4}\.\d{3})s \(kernel\) \+ \d{1,4}\.\d{3}s \(initrd\) \+ \d{1,4}\.\d{3}s \(userspace\) = (\d{1,4}\.\d{3})s/;
+        die 'Kernel boot is too slow - ' . $1          if $1 > $kernel_max_boot_time;
+        die 'Overall system boot is too slow  - ' . $2 if $2 > $system_max_boot_time;
```

Comment:
For me it is ok to use the log file from ipa, to get that information's.
We should add a extra threshold for wicked startup time of 30s, so we catch wicked timeouts.